### ISSUE TITLE
Obtain public developer application for the Okta source connector

### ISSUE THREAD

**Issue by igrankova** [airbytehq/airbyte#6186]
TBD

**Comment by igrankova** [comment]
[EPIC] Oauth developer applications #6070

**Comment by igrankova** [comment]
LastPass cred name: okta.com dev
Public App is in draft.
The OAuth flow passed successful.

**Comment by igrankova** [comment]
Public App is submitted for review on 04.11.2021

**Comment by YowanR** [comment]
@igrankova Can you share some details on the issues happening here?

**Comment by igrankova** [comment]
@YowanR, hi!
This source required OAuth and we submitted the public app, but to pass the process we needed the OAuth to be already implemented to record a video and test the app on the Okta side. We worked with the other team at that time who implemented OAuth and then the priorities changed and the teams switched to the certification process for more high priority connectors and Okta was temporary suspended. 
Now we are back to Okta and the certification is on the way. I need to continue the publishing. I see that Okta is not working on the Cloud and Anna Lvova is fixing it. Then I need to record a video how to use Okta connector. 
Later I'll ask you for help with the cloud test creds for the Okta support team - I already have them but there're no credits on this account and I'm afraid they won't be able to test the app - I'll let you know, we'll need to provide this test account for the temporary use. 
And there's one more trouble - after a long time passed I have no more access to the submission form, I'm going to resolve this with the Okta support.

**Comment by YowanR** [comment]
Thanks, @igrankova! Please keep me in the loop and let me know where I can help!

**Comment by YowanR** [comment]
Hi @igrankova, just checking in on this issue. Any updates? I've added you on a test account so we can try testing internally before sharing with Okta.

**Comment by igrankova** [comment]
Hi, @YowanR 
Yesterday we created a ticket to resolve a problem with sync Okta to Google Sheets: https://github.com/airbytehq/oncall/issues/380
The failure of the sync will be resolved by the PR: https://github.com/airbytehq/airbyte/pull/15179 
But the other part with the Resync button is still under the discussion.

Okta support unblocked our Listing Form and I've updated everything we missed during the downtime. I'm waiting for the PR merge to continue testing. 
According to the test creds, they gave us the requirements what email we should add for them: isvtest@okta.com. I've created this account to our Airbyte Cloud long time ago, it is live, just without the credits. I think we could activate it for the testing if they ask us. As we fix the sync I'll immediately submit the form for review.

**Comment by bazarnov** [comment]
https://github.com/airbytehq/airbyte/pull/15179 is closed now.

**Comment by igrankova** [comment]
Hi, @YowanR,
I've tested the connection on the cloud, everything works properly. I'm and ready to submit the form for review and publication of our Okta app. The last thing I need is to set up the test creds for the Okta support team. Can you help with this? 
Maybe we can add creds or somehow activate the test profile I've prepared: 
LastPass name **Okta App listing account**
User: isvtest@okta.com
![image](https://user-images.githubusercontent.com/87977540/182593912-03366def-bd08-49b3-b6d1-3c6d4944fb4e.png)

We have a requirement in the form to provide the temporary creds to our platform to test app on Okta side:  
![image](https://user-images.githubusercontent.com/87977540/182593406-7b0ad4a0-30ea-430d-940f-60bf1816e7ba.png)

**Comment by YowanR** [comment]
Hi @igrankova, I see that you've made a test connection in the Okta Partner Account that I shared with you. I checked and the connection works as expected. Can you confirm that you had credits to use on the accounts where you created the test connection? I can simply add isvtest@okta.com](mailto:isvtest@okta.com) to the account and they should be able to use it. Since you went through the flow once, can you let them know what to expect?

**Comment by igrankova** [comment]
Hi, @YowanR,
Yes, thank you, I've tested the connection on the account you gave and it works good. Sorry, I didn't understand that this account was created especially for Okta:) It would be great if you add their email to this account, as I remember they wanted to have that email as a test cred. 
The instructions how to test our connection I've added to the App Integration form, Additional Instructions field.

**Comment by YowanR** [comment]
Thanks @igrankova ! I've sent an invite to Okta (isvtest@okta.com). Please let me know if there's anything else I can do to help here :D

**Comment by YowanR** [comment]
What's the latest on this one, please? Still waiting for Okta to get back to us? @igrankova

**Comment by igrankova** [comment]
@YowanR yes, still no reply. I'm going to send them email and ask, I'll let you know if they reply.

**Comment by YowanR** [comment]
Did we ever hear back from Okta? @igrankova

**Comment by igrankova** [comment]
@YowanR, No:( The last I heard was:
![image](https://user-images.githubusercontent.com/87977540/187367422-d5195dcd-1cae-4017-8e3d-5052d951dbde.png)

Our App Status is:
![image](https://user-images.githubusercontent.com/87977540/187367209-3568ec90-7906-4ca8-9133-c6ba4a8a432c.png)

**Comment by YowanR** [comment]
Hi @igrankova @ycherniaiev Could you check the application for an update? I couldn't log in on my side.

**Comment by igrankova** [comment]
@YowanR, hi!
It's a bit tricky to login :)
First I log into the account: https://dev-01177082.okta.com/
![image](https://user-images.githubusercontent.com/87977540/190966042-96f77ec0-4938-4bfd-978f-16979e4ca40c.png)
Then - to the OIN, selecting our account: https://oinmanager.okta.com/
![image](https://user-images.githubusercontent.com/87977540/190966259-395a89a4-59a8-4fd4-8195-cc0496a1df20.png)
This is the status:

**Comment by igrankova** [comment]
The OIN requirements do not fit our solution.

https://developer.okta.com/docs/guides/submit-app/openidconnect/main/#protocols-supported 
<img width="763" alt="1" src="https://user-images.githubusercontent.com/87977540/210526709-abb5b810-4f8f-4dfc-bd6e-92fce177b606.png">

https://developer.okta.com/docs/guides/oin-oidc-protocols/main/#scopes 
<img width="670" alt="3" src="https://user-images.githubusercontent.com/87977540/210526745-770d0ddc-323e-40ee-b802-750a01062586.png">

https://developer.okta.com/docs/guides/oin-oidc-protocols/main/  
<img width="762" alt="2" src="https://user-images.githubusercontent.com/87977540/210526780-47e0bd2e-fd2f-4ff0-b160-6b65f99cf96a.png">
